modules/ROOT/pages/iam-limits.adoc

```diff
@@ -0,0 +1,24 @@
+= Limits in Access Management
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Access Management imposes the following limits on various features in an organization:
+
+* Roles per client: 3000
+* Clients per root organization: 10000
+* Client management providers per root organization: 10
+* Environments per organization: 150
+* Sub organizations per root organization: 100
+* Roles per organization: 500
+* External access relations per organization: 20
+* Teams per root organization: 1000
+* Users per organization: 10000
+* Roles per role group: 400
+* Roles per user: 5000
+* Role groups per user: 700
+* Connected apps per organization: 200
```

Comment:
Where is this defined in the code/config? I don't see 200 as a value here:
```
authentication-server git:(integration/develop) git grep CONNECTED api/config/external_properties.env | grep COLLEC
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_CONNECTEDAPPLICATIONS_SCOPES_ENABLED=[boolean:true]
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_CONNECTEDAPPLICATIONS_SCOPES_SLATIERS_LIMIT=[number:1000]
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_ORGANIZATIONS_CONNECTEDAPPLICATIONS_INTERNAL_ENABLED=[boolean:true]
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_ORGANIZATIONS_CONNECTEDAPPLICATIONS_INTERNAL_SLATIERS_LIMIT=[number:2000]
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_ORGANIZATIONS_CONNECTEDAPPLICATIONS_PUBLIC_ENABLED=[boolean:true]
api/config/external_properties.env:AUTH_SERVER_COLLECTION_LIMITING_POLICIES_ORGANIZATIONS_CONNECTEDAPPLICATIONS_PUBLIC_SLATIERS_LIMIT=[number:20]

➜  authentication-server git:(integration/develop) git grep 200\] api/config/external_properties.env
api/config/external_properties.env:AUTH_SERVER_MIGRATIONS_20190126166006_USER_ROLES_POPULATE_JSONB_TARGETTIME=[number:1200]
api/config/external_properties.env:AUTH_SERVER_MIGRATIONS_MASTERORGTEAMSMIGRATION_BATCHSIZE=[number:200]
api/config/external_properties.env:AUTH_SERVER_RATELIMITING_SLATIERS_CLIENTS_FIND_CALLERROUTE_INTERNAL_MAXREQUESTSPERWINDOW=[number:200]
api/config/external_properties.env:AUTH_SERVER_RATELIMITING_SLATIERS_IPROUTECHEAP_EXTERNAL_MAXREQUESTSPERWINDOW=[number:200]
api/config/external_properties.env:AUTH_SERVER_RATELIMITING_SLATIERS_ORGANIZATIONS_GETHIERARCHY_CALLERROUTE_INTERNAL_MAXREQUESTSPERWINDOW=[number:200]
```

I checked kilonova and there are no overrides (yay).

cf. https://mulesoft.slack.com/archives/C025U9CR49W/p1636652826015000?thread_ts=1636630156.008800&cid=C025U9CR49W

Reply:
https://mulesoft.slack.com/archives/C02FTAMUK9Q/p1637630854088800?thread_ts=1637630002.088700&cid=C02FTAMUK9Q